Xamarin.Forms.Platform.Android/Renderers/FormsEditText.cs

```diff
@@ -6,11 +6,11 @@
 
 namespace Xamarin.Forms.Platform.Android
 {
-	public class EntryEditText : EditText, IDescendantFocusToggler
+	public class FormsEditText : EditText, IDescendantFocusToggler
```

Comment:
Because it's used for both Entry and Editor.
Previously there were two classes (EntryEditText and EditorEditText) which were identical except for a transposed event name. Consolidating them makes things simpler and JITs fewer classes.